src/dfu.cpp

```diff
@@ -162,3 +188,23 @@ void dfu_nuke() {
     // Perform a hard reset
     asm volatile (" jmp 0");
 }
+
+void dfu_activate() {
+    update();
```

Comment:
Does this compile? I don't think we have this function... Also the activate function should probably be calling things like `node.begin()` (check how the node starts in normal mode)